src/CoreShop/Bundle/CoreBundle/Pimcore/Repository/CategoryRepository.php

```diff
@@ -53,18 +62,28 @@ public function findChildCategoriesForStore(CategoryInterface $category, StoreIn
      * {@inheritdoc}
      */
     public function findRecuriveChildCategoriesForStore(CategoryInterface $category, StoreInterface $store)
+    {
```

Comment:
`findRecuriveChildCategoriesForStore()` isn't available in current master.

Reply:
cause it got removed by another PR